pages/event-types/[type].tsx

```diff
@@ -778,7 +781,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                           </div>
                         </div>
                       </div>
-                      {web3App && (
+                      {data?.web3App && (
```

Comment:
very clean wrapper too. exactly how I thought about this

Reply:
the only thing I was wondering, is it worth it to rename `data?.web3App` to `app.web3App` when it's an App?